### PR TITLE
docs(readme): standardize

### PR DESCRIPTION
**Pull request by michael-ciniawsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,115 +1,101 @@
...
+logger.info('Server Starting')
 ```
 
-The code above will produce:
+![output](docs/output.png)
+
+> ℹ️ The logger returned is unique by default, due to the nature of the `webpack` ecosystem. Please reference the [`unique`](#unique) option below for disabling this feature and to **force caching**
 
-![output](output.png)
+<h2 align="center">Options</h2>
```
Comment (alexander-akait) [inline README.md]: Let's don't use `html` in markdown for headers, it is very bad, just use `## Options`

File: README.md [hunk 2]
```diff
@@ -1,115 +1,101 @@
...
 
-## Options
+|Name|Type|Default|Description|
+|:--:|:--:|:-----:|:----------|
+|[**`name`**](#name)|`{String}`|`''<unknown>'`|Log Name (**Required**)|
+|[**`level`**](#level)|`{String}`|`'info'`|Log Level|
+|[**`unique`**](#unique)|`{String}`|`true`|Log Uniqueness|
+|[**`timestamp`**](#timestamp)|`{Boolean}`|`false`|Log Timestamps|
```
Comment (alexander-akait) [inline README.md:46]: /cc @montogeek we use table?

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Looks very good, some fixes

File: README.md (hunk 2 above)
Comment (montogeek) [reply README.md:46]: Yes

**Pull request merged**